changelog/8.6.0.yaml

```diff
@@ -0,0 +1,283 @@
+version: 8.6.0
+entries:
+    - kind: feature
+      summary: Only support elasticsearch as an output for the beta synthetics integration.
+      description: ""
+      component: synthetics-integration
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1491
+      issue: []
+      timestamp: 1665780486
+      file:
+        name: 1665780486-heartbeat-es-output-only.yaml
+        checksum: 361f993ba3220aef531f7949308e61d2cbf82e03
+    - kind: feature
+      summary: Service runtime for V2
+      description: Service runtime for V2, tailored specifically for Endpoint service.
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1529
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1069
+      timestamp: 1666095433
+      file:
+        name: 1666095433-service_runtime_v2.yaml
+        checksum: 94f777454bf571caf8da848441c43eb6d7b3bc10
```

Comment:
```suggestion
```

Not a user visible change, internal re-architecture only. If we keep this we need to reword it not to mention V2 since users don't know what that is externally.